migrating/checklist/ossm-migrating-premigration-checklist-assembly.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-read-me-assembly.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklist-assembly]
+= Migrating from 2.6.z to 3.0 premigration checklist
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-migration-guides-assembly
+
+toc::[]
+
+Before you start migrating your `controlplane` from {SMProductName} 2.6 to 3.0, do the following:
+
+//Content is taken from here: https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/README.md
+//01/20/2025: there are change coming that link to 2.x content that users need to follow before starting to migrate from 2.x to 3. The engineering PR with these updates has not been merged into /openshift-service-mesh/sail-operator/docs/ossm/
+//check for updates at https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/README.md
+
+* [ ] Read "Important information to know if you are migrating from OpenShift Service Mesh 2.6".
+* [ ] Upgrade the {SMProduct} Operator to the latest 2.6.z release.
+* [ ] Upgrade your `ServiceMeshControlPlane` version to the latest {SMProduct} 2.6.z release.
+
+== Disable addons on your `ServiceMeshControlPlane`
+
+These addons are not supported in {SMProduct} 3.0 and must be disabled before migrating.
+
+** [ ] Disable network policy management by setting `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.6 ServiceMeshControlPlane, you will need to recreate and manage these manually.
+** [ ] Disable Prometheus by setting `spec.addons.prometheus.enabled=false`. See "User workload monitoring" for instructions configuring {SMProduct}h 3.0 with OpenShift Monitoring as a replacement for Prometheus.
+** [ ] Disable Kiali by setting `spec.addons.kiali.enabled=false``. See "Kiali Operator provided by Red Hat" for instructions installing and configuring {KialiProduct} with {SMProduct} 3.0.
+** [ ] Disable Grafana by setting `spec.addons.grafana.enabled=false`.
+** [ ] Disable {DTProductName} by setting `spec.tracing.type=None`. See "{DTProductName}" for instructions on configuring {SMProduct} 3.0 with {DTProductName}.
+IOR is disabled.
+** [ ] Disable default ingress and egress gateways.
+
+== Check what type of mesh you are running
+
+Run the following command to determine if you are running a cluster-wide or multitant mesh:
+
+[source,terminal]
+----
+$ oc get smcp <smcp-name> -n <smcp-namespace> -o jsonpath='{.spec.mode}'
+----
+
+//01/10/2024: still need instructions from dev on cluster-wide
+
+.Next steps for multitenant deployments
+
+For multitenant deployments, follow this procedure:
+//Applicable PR: https://github.com/openshift/openshift-docs/pull/86917
+//will contain both mulitenant and cert-manager procedures
+
+.Next steps for multitenant deployments with cert-manager
```

Comment:
Not sure if this affects how you structure things or not but the migration guide for cert-manager is essentially the same for both `MultiTenant` and `ClusterWide`. The only difference is that some `MultiTenant` deployments might be using a specific field/feature of cert-manager in which case they will need to handle it differently than `ClusterWide`.

Reply:
I'm remembering now we discussed that some duplication was better than having the guide more scattered